### DISCUSSION TITLE
Photoprism CLI creates no output

### DISCUSSION THREAD

**Discussion by vsaw** [photoprism/photoprism#3585] [Q&A]
Im using Photoprism via OpenMediaVault. When trying to access the CLI, I get no output from Photoprism. Here's an example command I tried to run

```
podman exec photoprism-app photoprism help
```

I then don't see any output. The command just hangs until I cancel it using `CTRL + C`. On the other hand, things like `podman exec photoprism-app ls` do list directory contents so I expect podman to work correctly. 

Do I need to consider something else to use the CLI with podman/OpenMediaVault?

**Comment by graciousgrey** [comment]
In the [OMV forum](https://forum.openmediavault.org/index.php?thread/43757-photoprism-plugin-cli-commands/) a user recommends using commands like this `sudo podman exec -ti photoprism-app photoprism help` , maybe this works?
